clearpath_config/platform/battery.py

```diff
@@ -54,6 +54,7 @@ class BatteryConfig(BaseConfig):
     NEC_ALM12V35 = "NEC_ALM12V35"
     VALENCE_U24_12XP = "VALENCE_U24_12XP"
     VALENCE_U27_12XP = "VALENCE_U27_12XP"
+    INVENTUS_S_24V20_U1 = "S_24V20_U1"
```

Comment:
In the `clearpath_robot` package, the battery is name `BatteryConfig.S_24V20_U1` not `BatteryConfig.INVENTUS_S_24V20_UI`.